src/Event.js

```diff
@@ -0,0 +1,46 @@
+import * as Utility from './Utility.js'
+import * as Lifecycle from './Lifecycle.js'
+import * as Schedule from './Schedule.js'
+import * as Interface from './Interface.js'
+
+/**
+ * @param {object} event
+ */
+export function handle (event) {
+	dispatch(this.host, event, Interface.event(event, this.instance))
+}
+
+/**
+ * @param {object} host
+ * @param {object} event
+ * @param {*} callback
+ */
+export function dispatch (host, event, callback) {
+	Schedule.checkout(enqueue, host, event, callback)
+}
+
+/**
+ * @param {object} fiber
+ * @param {object} host
+ * @param {object} event
+ * @param {*} callback
+ */
+export function enqueue (fiber, host, event, callback) {
+	if (callback) {
```

Comment:
should be unnecessary (invariant?)
Also couldn't find outside usage of any other exported function (besides `handle`) (or is the pattern to export everything always?)

Reply:
This handles the case when events are undefined i.e `h('div', {onClick: null})`. 

Yes i think there's is no harm in always exporting, it's the same reason we use `import * as Name from 'file.js'`: because rollup is sufficiently smart enough to handle these patterns.

Right now the bundle is at around ~5kb.